stat/distuv/uniform_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2016 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math/rand"
+	"sort"
+	"testing"
+
+	"gonum.org/v1/gonum/floats"
+)
+
+func TestUniformProb(t *testing.T) {
+	for _, test := range []struct {
+		min, max, x, want float64
+	}{
+		{0, 1, 1, 1},
+		{2, 4, 0, 0},
+		{2, 4, 5, 0},
+		{2, 4, 3, 0.5},
+		{0, 100, 1, 0.01},
+	} {
+		pdf := Uniform{test.min, test.max, nil}.Prob(test.x)
+		if !floats.EqualWithinAbsOrRel(pdf, test.want, 1e-10, 1e-10) {
```

Comment:
These tolerances should be higher, there basically shouldn't be error. 1e-15 instead

Reply:
done